metagraph/src/kmer/kmer_extractor.cpp

```diff
@@ -399,6 +399,44 @@ void KmerExtractorBOSS::sequence_to_kmers(std::string_view,
                                           Vector<Kmer256>*,
                                           bool);
 
+template <typename KMER>
+KMER KmerExtractorBOSS::sequence_to_kmer(std::string_view sequence,
+                                         bool canonical_mode) {
+    // encode sequence
+    std::vector<TAlphabet> seq(sequence.size(), alphabet.size());
```

Comment:
```suggestion
    std::vector<TAlphabet> seq(sequence.size());
```

Reply:
You reinitialize everything in the next line, there is no point to set the initial value.